tests/unit/utils/test_stringutils.py

```diff
@@ -91,6 +94,23 @@ def test_to_bytes(self):
             self.assertEqual(salt.utils.stringutils.to_bytes(UNICODE, 'utf-8'), BYTES)
 
     def test_to_unicode(self):
+        self.assertEqual(
+            salt.utils.stringutils.to_unicode(
+                EGGS,
+                encoding='utf=8',
+                normalize=True
+            ),
+            'яйца'
+        )
+        self.assertNotEqual(
+            salt.utils.stringutils.to_unicode(
+                EGGS,
+                encoding='utf=8',
+                normalize=False
+            ),
+            'яйца'
+        )
+
         if six.PY3:
```

Comment:
This and below seems no longer relevant.

Reply:
At least part of it is relevant because it tests python2-specific code, but most of it can be combined in a single section.

Reply:
I will open a separate PR for that.